ui/pages/send/send.component.js

```diff
@@ -171,7 +171,7 @@ export default class SendTransactionScreen extends Component {
     if (qrCodeData) {
       if (qrCodeData.type === 'address') {
         scannedAddress = qrCodeData.values.address.toLowerCase();
-        if (isValidAddress(scannedAddress)) {
+        if (isValidHexAddress(scannedAddress)) {
```

Comment:
This should be following the B strategy too I think?

Reply:
https://github.com/MetaMask/metamask-extension/blob/v9.4.0/ui/app/pages/send/send.component.js#L174

Reply:
Whoops! So should we use strategy A here instead?

Reply:
Huh, thought I did 🤦🏻  done now!